test/distributed/fsdp/test_fsdp_use_orig_params.py

```diff
@@ -411,9 +411,9 @@ def _test_multiple_optimizers(self, sharding_strategy: ShardingStrategy):
                 has_weight = False
                 has_bias = False
                 for param, fqn in zip(flat_param._params, flat_param._fqns):
-                    if "weight" in fqn and param.numel() > 0:
+                    if fqn is not None and "weight" in fqn and param.numel() > 0:
```

Comment:
Is there any way _params and _fqns could just not return these padding parameters which aren't real parameters? i.e. have a `@property` accessor that filters these out?